### PR TITLE
Expose Management Plane UIs in Console UI for Customer Admin role

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
[1535](https://github.com/kyma-project/console/issues/1535)
[7210](https://github.com/kyma-project/kyma/issues/7210)
[Console](https://github.com/kyma-project/console/pull/1558)
[Compass](https://github.com/kyma-incubator/compass/pull/830)

### CODE REVIEW

File: resources/application-connector/templates/applications-microfrontend.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.managementPlaneUrl -}}
```
Comment (kwiatekus) [inline resources/application-connector/templates/applications-microfrontend.yaml]:
```suggestion
{{- if .Values.global.managementPlane.url  -}}
```

File: resources/compass/charts/cockpit/templates/compass-scenarios-and-runtimes-microfrontend.yaml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.managementPlaneUrl -}}
```
Comment (kwiatekus) [inline resources/compass/charts/cockpit/templates/compass-scenarios-and-runtimes-microfrontend.yaml]:
```suggestion
{{- if .Values.global.managementPlane.url -}}
```

File: resources/core/charts/console/templates/configmap.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {{- $embeddedCompassApiUrl :=  printf "https://compass-gateway.%s/director/graphql" .Values.global.ingress.domainName }}
-{{- $compassApiUrl := .Values.cluster.managementPlaneApiUrl | default $embeddedCompassApiUrl -}}
+{{- $compassApiUrl := .Values.global.managementPlaneUrl | default $embeddedCompassApiUrl -}}
```
Comment (kwiatekus) [inline resources/core/charts/console/templates/configmap.yaml]:
```suggestion
{{- $compassApiUrl := .Values.global.managementPlane.url | default $embeddedCompassApiUrl -}}
```

File: resources/core/charts/console/templates/test/test.yaml [hunk 4]
```diff
@@ -46,6 +46,8 @@ spec:
                    memory: {{ .Values.test.acceptance.ui.limits.memory }}
                    cpu: {{ .Values.test.acceptance.ui.limits.cpu }}
               env:
+                - name: MANAGEMENT_PLANE_URL
+                  value: {{ .Values.global.managementPlaneUrl | quote }}
```
Comment (kwiatekus) [inline resources/core/charts/console/templates/test/test.yaml]:
```suggestion
                  value: {{ .Values.global.managementPlane.url | quote }}
```

File: resources/application-connector/templates/applications-microfrontend.yaml (hunk 1 above)
Comment (Wawrzyn321) [reply resources/application-connector/templates/applications-microfrontend.yaml]: All changes to this file have been reverted. Resolving.

**Comment by kwiatekus** [comment]
@pbochynski Done.
Separate PR with code owners changes here: 
https://github.com/kyma-project/kyma/pull/7284/files

File: resources/compass-runtime-agent/values.yaml [hunk 5]
```diff
@@ -12,6 +12,8 @@ global:
       dir: "develop/"
       version: "8381946f"
 
+managementPlane: {} # default value
```
Comment (dariadomagala) [inline resources/compass-runtime-agent/values.yaml]: Why do we need it in two places?

**Pull request merged**